src/sentry/lang/javascript/processor.py

```diff
@@ -511,6 +511,9 @@ def fetch_release_artifact(url, release, dist):
         try:
             archive = ReleaseArchive(archive_file)
         except Exception as exc:
+            archive_file.seek(0)
+            # Assign to variable to make contents visible in Sentry:
+            contents = archive_file.read(n=256)  # NOQA
```

Comment:
nit: This will work but maybe cleaner would be to use logging extra argument. e.g. https://github.com/getsentry/sentry/blob/master/src/sentry/tasks/store.py#L113

Reply:
Today I learned! That's nicer indeed 👍